### PR TITLE
updated the docs

### PR DESCRIPTION
**Pull request by voroyam**
After a request from @pako81 I have update the docs to fix https://github.com/owncloud/docs/issues/1984

### CODE REVIEW

File: modules/admin_manual/pages/maintenance/upgrade.adoc [hunk 1]
```diff
@@ -49,4 +49,4 @@ in conjunction with our official ownCloud repositories. *Note:* This approach sh
 nor in clustered setups.
 We discourage upgrades with Linux Package Manager because you might encounter unwanted side effects.
 
-NOTE: Enterprise customers will use their Enterprise software repositories to maintain their ownCloud servers, rather than the Open Build Service. Please see xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.
+NOTE: Enterprise customers will refer to Installing & Upgrading ownCloud Enterprise Edition for more information. Please see xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.
```
Comment (pako81) [inline modules/admin_manual/pages/maintenance/upgrade.adoc]: Redundant info here? It should be:

NOTE: Enterprise customers will refer to xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.

**Comment by settermjd** [comment]
@voroyam, this looks good to me, after a minor reduction in content.

**Comment by pako81** [comment]
Already fixed :) thanks @settermjd @voroyam :)

File: modules/admin_manual/pages/maintenance/upgrade.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/maintenance/upgrade.adoc]: Agreed.

**Pull request merged**